packages/usa-paragraph/src/styles/_usa-paragraph.scss

```diff
@@ -1,9 +1,9 @@
 @use "uswds-core" as *;
 
-%usa-paragraph {
-  @include typeset-p;
-}
-
 .usa-paragraph {
   @include typeset-p;
+
+  * + & {
+    margin-top: 1em; // TODO: add as var to settings?
+  }
```

Comment:
`* + &` gets confused when added to mixins, so moved these items directly inside the style rules.